hmac/play/play_2-9/src/test/scala/com/gu/pandahmac/HmacSecretsTest.scala

```diff
@@ -0,0 +1,60 @@
+package com.gu.pandahmac
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import java.net.URI
+import java.time.format.DateTimeFormatter
+import java.time.{Clock, Instant, ZoneId}
+
+class HMACHeadersTest extends AnyWordSpec with Matchers {
```

Comment:
Ooh, new in this PR? 👏

Reply:
> Ooh, new in this PR?

Ah, I'm afraid not - I accidentally made two copies of the same test file! I've removed the second copy with 9af8d90.